sdk/apimanagement/Azure.ResourceManager.ApiManagement/src/autorest.md

```diff
@@ -313,6 +313,25 @@ directive:
       $.PortalSettingsContractProperties.properties.subscriptions['x-ms-client-name'] = 'IsSubscriptions';
       $.PortalSettingsContractProperties.properties.userRegistration['x-ms-client-name'] = 'IsUserRegistration';
       $.PrivateEndpointConnectionRequest.properties.id['x-ms-format'] = 'arm-id';
+      $.AccessInformationUpdateParameterProperties.properties.enabled['x-ms-client-name'] = 'IsEnabled';
+      $.AuthenticationSettingsContract.properties.oAuth2["x-nullable"] = true;
+      $.AuthenticationSettingsContract.properties.openid["x-nullable"] = true;
+      $.ApiEntityBaseContract.properties.authenticationSettings["x-nullable"] = true;
+      $.ApiEntityBaseContract.properties.subscriptionKeyParameterNames["x-nullable"] = true;
+      $.LoggerContractProperties.properties.resourceId["x-nullable"] = true;
+      $.ApiContractProperties.properties.protocols["x-nullable"] = true;
+      $.ApiTagResourceContractProperties.properties.protocols["x-nullable"] = true;
+      $.ApiContractUpdateProperties.properties.protocols["x-nullable"] = true;
+      $.NamedValueEntityBaseParameters.properties.tags["x-nullable"] = true;
+      $.SubscriptionContractProperties.properties.startDate["x-nullable"] = true;
+      $.SubscriptionContractProperties.properties.expirationDate["x-nullable"] = true;
+      $.SubscriptionContractProperties.properties.endDate["x-nullable"] = true;
+      $.SubscriptionContractProperties.properties.notificationDate["x-nullable"] = true;
+      $.PortalSettingsContractProperties.properties.termsOfService["x-nullable"] = true;
+      $.PolicyDescriptionContractProperties.properties.scope = {
+          "description": "Binary OR value of the Snippet scope.",
+          "readOnly": true
+        }
```

Comment:
Double confirm: we make this an "anyObject" on purpose based on test cases?
If so please also add some comments on this directive (maybe with a reference to the name of the test case that need this)